infra/aws/terraform/prow-build-cluster/kubernetes.tf

```diff
@@ -40,6 +40,39 @@ module "metrics_server" {
   ]
 }
 
+# AWS Load Balancer Controller (ALB/NLB integration).
+resource "helm_release" "aws_lb_controller" {
+  name       = "aws-load-balancer-controller"
+  namespace  = "kube-system"
```

Comment:
I usually put the ALB load balanced controller into a less privileged namespace.